ui/page/dexclient/markets.go

```diff
@@ -42,10 +54,14 @@ func (pg *Page) dexMarket(mktName string) *core.Market {
 
 func NewMarketPage(l *load.Load) *Page {
 	pg := &Page{
-		Load:       l,
-		login:      l.Theme.Button("Login"),
-		initialize: l.Theme.Button("Start using now"),
-		addDex:     l.Theme.Button("Add a dex"),
+		Load:          l,
+		login:         l.Theme.Button("Login"),
+		initialize:    l.Theme.Button("Start using now"),
+		addDex:        l.Theme.Button("Add a dex"),
+		sync:          l.Theme.Button("Start sync to continue"),
+		toWallets:     l.Theme.Button("Settings Wallets"),
```

Comment:
`toWallets` should be `walletSettingsButton`
"Settings Wallets" should be "Wallet Settings"

Reply:
Yh, walletSettings is ok. don't need Button at the end